src/components/header/Header.module.css

```diff
@@ -173,6 +176,46 @@
 }
 
 @media (max-width: 991px) {
+  .header {
+    padding-top: var(--pt1);
+    padding-bottom: var(--pt1);
+    height: var(--pt9);
+  }
```

Comment:
Jeg ville personligt splitte media-queries op, og placere f.eks. `.header {}` og `@media (....) { .header {} }` sammen, men du kan bare ignorere denne kommentar hvis du er ligeglad

Reply:
.. taler lidt for scss. enig, men denne fil var allerede lavet med en samlet media query. - bedre at være konsvekvent her, og gad ikke refaktor det hele